app/views/candidate_interface/application_form/start_apply_again.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1 class="govuk-heading-xl">Do you want to apply again?</h1>
```

Comment:
Looks like this page might be missing a page title (`<title>`).

Reply:
thanks @fofr - I've added a `title` in a follow up PR https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1995